Makefile

```diff
@@ -11,27 +11,30 @@ DISTDIR   := $(TARDIR)/dist
 TARNAME   := $(TARDIR).tar.gz
 APP_DIST  := ./dist
 
-build-all: build-setup build
+build-all: setup compile tar
```

Comment:
Can we create the wrapper params as `build-tar` and `build-rpm`?
This is pretty easy to do, with the following code, by just maintaining the procedural flow.